boards.txt

```diff
@@ -17,6 +17,7 @@ menu.LoRaWanDebugLevel=LoRaWan Debug Level
 menu.LoopCore=Arduino Runs On
 menu.EventsCore=Events Run On
 menu.MemoryType=Memory Type
+menu.EraseFlash=Erase Flash (on upload)
```

Comment:
Would it be more clear if the menu just says: 

"Erase all Flash before uploading the sketch:" "Yes" or "No" ?

At the end this is just an single option.